netbox/resource_netbox_device_test.go

```diff
@@ -108,6 +113,7 @@ resource "netbox_device" "test" {
   site_id = netbox_site.test.id
   cluster_id = netbox_cluster.test.id
   location_id = netbox_location.test.id
+	config_template_id = netbox_config_template.test.id
```

Comment:
indentation looks weird on github. Probably spaces vs tabs (use what the surrounding code uses, probably spaces)